testplan/testing/multitest/driver/tcp/client.py

```diff
@@ -167,7 +167,11 @@ def starting(self):
         """Start the TCP client and optionally connect to host/post."""
         super(TCPClient, self).starting()
         self._server_host = expand(self.cfg.host, self.context)
-        self._server_port = expand(self.cfg.port, self.context, int)
+        self._server_port = (
+            expand(self.cfg.port, self.context, int)
```

Comment:
the expanded port might be a string as well, need to transform that

Reply:
Thanks for the feedback @kn-ms 
The third param will be used as the constructor. Please check the implementation of the expand

Reply:
Okay got it now. Thanks for the clarification